src/utils/handleSubmit.js

```diff
@@ -0,0 +1,37 @@
+import Popup from "../components/Popup.js";
+
+export function handleSubmit(
```

Comment:
Actually, `handleSubmit` is not needed anymore, because you've implemented everything inside the classes. You have `then`, `catch` and `finally` inside `PopupDeleteCard` and `PopupWithForm`